guides/01_getting-started/02_key-features.md

```diff
@@ -258,6 +258,8 @@ demo.launch()
 
 In the example above, 16 requests could be processed in parallel (for a total inference
 time of 5 seconds), instead of each request being processed separately (for a total
-inference time of 80 seconds).
+inference time of 80 seconds). Many Hugging Face `transformers` and `diffusers` models
+work very naturally with Gradio's batch mode: here's [an example demo using diffusers to
+generate images in batches](https://github.com/gradio-app/gradio/blob/main/demo/diffusers_with_batching/run.py)
```

Comment:
Why aren't we embedding this demo anywhere?

Reply:
I guess looking at the demo the usage itself doesn't show anything? So up to you

Reply:
Mostly because I felt it was taking up too much space already

Reply:
The demo itself doesn't change, mostly just the code usage